Rakefile

```diff
@@ -615,14 +615,6 @@ begin
       add_empty_master_changelog_section('.') && sh("git commit -am '[CHANGELOG] Add empty Master section'")
       sh "git push origin #{current_branch}"
       sh "git checkout master"
-      begin
```

Comment:
should print a comment reminding the releaser to merge back to stable

Reply:
sg

Reply:
updated